src/components/Wallet/index.jsx

```diff
@@ -6,8 +6,10 @@ import {
   validateBIP32Path,
   validateExtendedPublicKey,
   satoshisToBitcoins,
+  convertExtendedPublicKey,
 } from "unchained-bitcoin";
 import { Grid, Box, Button, FormHelperText } from "@material-ui/core";
+import { encodeUR } from "@cvbb/bc-ur";
```

Comment:
I think this is the only place it's used - but if we can bring it in via `unchained-wallets` then we don't have to maintain this package version in the caravan application.